File: CourseProjects/CourseProjects/VisualStudioProjects/ASP/AspWebFormsExamplesAndExercises/aspx_cs_samples/App_Code/CustomSettingsVO.cs
using System;

namespace AspxDemos {
	public class CustomSettingsVO {
		public CustomSettingsVO(int maxThreads, string databaseIP) {
			this.maxThreads = maxThreads;
			this.databaseIP = databaseIP;
		}

		public string DatabaseIP {
			get { return databaseIP; }
		}

		public int MaxThreads {
			get { return maxThreads; }
		}

		private readonly string databaseIP;
		private readonly int maxThreads;
	}
}

File: CourseProjects/CourseProjects/VisualStudioProjects/ASP/AspWebFormsExamplesAndExercises/aspx_vb_samples/App_Code/VisualBasicWebApp/CustomSettingsVO.vb
Imports Microsoft.VisualBasic

Public Class CustomSettingsVO
    Public Sub New(ByVal _maxThreads As String, ByVal _databaseIP As String)
        Me._databaseIP = _databaseIP
        Me._maxThreads = _maxThreads
    End Sub
    Public ReadOnly Property MaxThreads() As String
        Get
            Return _maxThreads
        End Get
    End Property
    Public ReadOnly Property DatabaseIP() As String
        Get
            Return _databaseIP
        End Get
    End Property
    Private _maxThreads As String
    Private _databaseIP As String
End Class
